tests/e2e/pom/Base.page.ts

```diff
@@ -1,32 +1,33 @@
-import { Page, Locator, expect} from '@playwright/test'
+import { Page, Locator, expect } from "@playwright/test";
 
 export class BasePage {
-    readonly page: Page;
-    readonly errorMessage: Locator;
-    readonly successMessage: Locator;
-    readonly pageHeading: Locator;
-    readonly logoutButton: Locator;
-    
+  readonly page: Page;
+  readonly errorAlert: Locator;
+  readonly successMessage: Locator;
+  readonly pageHeading: Locator;
+  readonly logoutButton: Locator;
+  readonly nextButton: Locator;
```

Comment:
In terms of generalization - it's ok.
But in terms of test - we will not notice, if label of the button is changed.

How could you solve this issue?